src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -979,41 +982,76 @@ private function registerValidationConfiguration(array $config, ContainerBuilder
         }
     }
 
-    private function getValidatorMappingFiles(ContainerBuilder $container)
+    /**
+     * Get all validation files.
```

Comment:
I think the whole phpdoc block can be removed. It does not add that much value (and is for a private method anyway).